### ISSUE TITLE
[Image][Accessibility] Empty alt attribute for non-decorative images

### ISSUE THREAD

**Issue by jakublamprecht** [adobe/aem-core-wcm-components#922]
## Bug Report

**Current Behavior**
Image component configured in a way that checkbox 'Get alternative text from DAM' is checked has an empty alt attribute in markup if title and description properties of used asset are empty (and Image component has checkbox 'Image is decorative' uncheked).

This behaviour can cause a false-positive result of accessibility testing with tools.

The image was not marked as decorative and doesn't provide required information to the used, which is a content issue that should be reported by accessibility testing tools.
Those tools will not report the issue however, since they detect the empty alt attribute in markup and treat the image as decorative.

**Expected behavior/code**
If author forgets to fill properties that are used for generating value of 'alt' attribute, the issue should be reported and no alt attribute should be added in markup.

**Environment**
- AEM version - AEM 6.5 SP3
- Core Components 2.8.0

**Comment by jakublamprecht** [comment]
There's also a similar issue to the one described in #919, where ALT is taken from properties even if it's configured to take ALT from DAM (when title and description of asset are empty).

Do you think this should be addressed in this issue, or would you rather I created a separate one?

**Comment by bpauli** [comment]
@jakublamprecht Thanks you for the hands up! The issue with the empty alt attribute for the image component should be fixed in the PR at https://github.com/adobe/aem-core-wcm-components/commit/753081a42f5499802c2fa0dc70c4a36b17979d40.

The issue reported in #919 for the Download component was a bit different. The title of the download component was taken from the component properties even the checkbox was checked to get the title from DAM asset. This false behaviour was fixed in the commit 31ea736d2211333d92daed2a4c0ad2e170edcfd4.

If you think that there is also an accessibility issue with the download component it would be nice if can create a new issue. Many thanks in advance!
